pyproject.toml

```diff
@@ -56,7 +56,7 @@ loguru = "^0.5.3"
 hydra-slayer = "^0.2.0"
 typer = "^0.4.0"
 omegaconf = "^2.1.1"
-holidays = ">=0.13,<1.0"
+holidays = ">=0.20,<1.0"
```

Comment:
You updated the version here, but you haven't updated `poetry.lock`. You could do it using the command `poetry lock --no-update`. After solving you could install from the new lock using `poetry install --extras all-dev"

Reply:
Do we really want to make this update? [Version 0.20 was released in February 2023](https://pypi.org/project/holidays/#history), it is very new.